protocol/x/clob/keeper/process_operations.go

```diff
@@ -75,6 +91,21 @@ func (k Keeper) ProcessProposerOperations(
 			allUpdates.Append(orderbookUpdate)
 		}
 		k.SendOrderbookUpdates(ctx, allUpdates)
+
+		// send subaccount snapshots
+		subaccountIdsFromProposed := fetchSubaccountIdsInvolvedInOpQueue(
```

Comment:
super meganit, up to you: combine the two loops where we iterate through the opqueue?

Reply:
only sending local for now